bot.js

```diff
@@ -88,6 +97,18 @@ client.on('message', async (msg) => {
 		return userinfo(msg, member, author)
 	}
 
+	if (msg.content.startsWith('$tzrefresh')) {
+		userroles = msg.member.roles.cache
+		elite_crew_id = "760665499330936922"
+		bossman_id = "553723628957728820"
+		if (userroles.find(r => r.id === elite_crew_id) || userroles.find(r => r.id === bossman_id)) {
+			logger("generated timezone list")
+			return memberstzs(msg)
+		} else {
+			return
+		}
```

Comment:
the else statement can leave, with return going at the end, since the function will get returned if the if comes back as true

Reply:
True, but I wasn't sure how well that would show the program logic. I can change it though if you want.